backend/tracim_backend/applications/agenda/application.py

```diff
@@ -31,6 +31,16 @@ def load_config(self, app_config: CFG) -> None:
             "caldav.radicale.storage.filesystem_folder", default_caldav_storage_dir
         )
 
+        app_config.CALDAV__PRE_FILLED_EVENT__DESCRIPTION_FILE_PATH = app_config.get_raw_config(
+            "caldav.pre_filled_event.description_file_path", "",
+        )
+
+        if app_config.CALDAV__PRE_FILLED_EVENT__DESCRIPTION_FILE_PATH:
+            with open(app_config.CALDAV__PRE_FILLED_EVENT__DESCRIPTION_FILE_PATH) as f:
+                app_config.CALDAV__PRE_FILLED_EVENT__DESCRIPTION = f.read()
```

Comment:
```suggestion
                app_config.CALDAV__PRE_FILLED_EVENT__DESCRIPTION = f.read()
```
What happened if file path doesn't exist ?

Reply:
if you have a way to distinct the "failed" and the no file path case without much trouble, i do suggest to add something in the check_config part to fail in this case with an explicity error, the file should exist and be readable.

Reply:
@raphj

Reply:
yes

Reply:
Can you approve?
@PhilippeAccorsi too